### ISSUE TITLE
Api Route Does Not Exist. 

### ISSUE THREAD

**Issue by svewa** [Ride-The-Lightning/RTL#1250]
**Describe the bug**
Most information is not available on the dashboard, channel list etc, "Api Route Does Not Exist." is shown instead.

**To Reproduce**
Open RTL

**Screenshots**
![grafik](https://github.com/Ride-The-Lightning/RTL/assets/20936492/148797d5-9290-43ed-85a2-d46ed2f30a1e)

**Your environment**
Everything just updated now, so I don't know if the rest-api is missing it, or if RTL is calling the wrong api?
CLN v23.05
API 0.10.3
RTL 0.14.0

**Additional context**
after updating RTL first, the channel balances did not show, rest was working (0 sats on both sides) so I updated the API...

**Comment by ShahanaFarooqui** [comment]
This error is consistent with old api version. Please check that you are not running the older version of c-lightning-rest.

**Comment by ShahanaFarooqui** [comment]
Yes, it is the correct version then. AFAICT it is due to listpeerchannels api call which has been introduced in CLN 23.05 and rest 0.10.3. With 0.10.3 it should not throw this error. I will try to reproduce it again but we already tested it on 3-4 nodes.
